features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
```

Comment:
Actually IMO the features description should be similar to:
`Apply taxes based on zone and tax rate`